Tests/Common/Orders/Fills/ImmediateFillModelTests.cs

```diff
@@ -282,29 +283,41 @@ public void PerformsStopMarketFillBuy()
                 new SecurityCache()
             );
             security.SetLocalTimeKeeper(TimeKeeper.GetLocalTimeKeeper(TimeZones.NewYork));
-            security.SetMarketPrice(new IndicatorDataPoint(Symbols.SPY, Noon, 101m));
+            var slippageModel = new ConstantSlippageModel(.01m);
+            security.SetSlippageModel(new ConstantSlippageModel(.01m));
```

Comment:
We don't need to create two instances of `ConstantSlippageModel`:
```csharp
security.SetSlippageModel(slippageModel);
```
Same [here](https://github.com/QuantConnect/Lean/pull/4546/files#diff-b29d6b609cdd690136095fe84443a4a5R341).

Reply:
agree